examples/named-worker-return-values/named_worker_return_values.bal

```diff
@@ -1,13 +1,13 @@
 import ballerina/io;
 
 function demo(string s) returns int|error {
-    // Named workers have a return type, which defaults to nil
+    // Named workers have a return type, which defaults to `nil`
```

Comment:
We don't have nil as a keyword, so we don't usually emphasize it.

We usually emphasize keywords and identifiers in the code.